test/Repos.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+require('chai').use(require('sinon-chai'));
```

Comment:
@ronanfitz rather than requiring `chai` twice here, please require it once (assigning it to the variable `chai`, and then assign `expect` to be `chai.expect` and then `chai.use(require('sinon-chai'));` This prevents us from having to load the entire library twice, which is potentially expensive. Please do this throughout the tests, thanks!